src/metabase/driver/mysql.clj

```diff
@@ -65,7 +65,8 @@
       kdb/mysql
       ;; 0000-00-00 dates are valid in MySQL, but JDBC barfs when queries return them because java.sql.Date doesn't allow it.
       ;; Add a param to the end of the connection string that tells MySQL to convert 0000-00-00 dates to NULL when returning them.
-      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull"))))
+      ;; Also add params to force UTF-8 encoding of results
+      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF8&characterSetResults=UTF8"))))
```

Comment:
If I understand this correctly it's just telling MySQL to return the results and interpret queries with UTF-8. I don't *know* if it is converting strings in other encodings to UTF-8 but I don't think it would otherwise make things any worse if that was the case :scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat::scream_cat: